.vscode/launch.json

```diff
@@ -33,6 +33,22 @@
             "preLaunchTask": "buildIdentityApiAdmin",
             "stopAll": true
         },
+        {
+            "name": "Admin, API, Identity, SSO",
+            "configurations": [
+                "run-Admin",
+                "run-API",
+                "run-Identity",
+                "run-Sso"
+            ],
+            "presentation": {
+                "hidden": false,
+                "group": "AA_compounds",
+                "order": 4
+            },
+            "preLaunchTask": "buildIdentityApiAdmin",
```

Comment:
I think we need a build task defined in `tasks.json` for this new configuration (see Matt's PR [here](https://github.com/bitwarden/server/pull/3446/files) to set this up).

Reply:
I addressed this as well as removed Admin in [`cdde050` (#4168)](https://github.com/bitwarden/server/pull/4168/commits/cdde050c544ace0d6cefdb404164689bf444c556).